idea_town/frontend/static-src/app/views/landing_page.js

```diff
@@ -0,0 +1,37 @@
+import app from 'ampersand-app';
+import AmpersandView from 'ampersand-view';
+import mustache from 'mustache';
+
+// TODO replace with an api endpoint that exposes idea town addon info
+const addonInfo = {
+  name: 'Idea Town',
+  url: '//localhost/codez/github/mozilla-idea-town-addon/@idea-town-addon-0.0.1.xpi'
```

Comment:
+1. This line didn't work for me. Not sure if we want to hardcode to something like this for the time bein:
https://github.com/mozilla/idea-town-addon/blob/master/dist/idea-town-addon-0.0.1.xpi?raw=true